capabilities/backend-development.html

```diff
@@ -0,0 +1,30 @@
+---
+layout: capability
+
+asset-name: backend_development
+
+description: "The stuff you never see is the stuff that makes your applications sing.
+We’re good at it, and we have examples to show you!"
+
+tags: [Back End Development]
+
+title: Back-End Development
+
+subtitle: "It gets no fanfare, but it does all the hard work invisibly."
+
+text: '<p>How it works. You store your data somewhere - likely the cloud - and then give
+users a way to interact with that data through APIs. That’s the basic gist of back-end.</p>
+<p>APIs are considered back-end development. The code that runs on the back end of the
+server. The front end runs on a browser. The API makes the call to the back-end. It
```

Comment:
Remove this sentence:
The API makes the call to the back-end.

Reply:
Yes

Reply:
Removed sentence plus removed 'The front end runs on a browser.'